packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -270,25 +270,25 @@ curl -s -XPUT -H "Content-Type: application/json" -H "Authorization: SSWS ${api_
 
 <ApiOperation method="delete" url="/api/v1/meta/types/user/${typeId}" />
 
-Deletes a User Type permanently. This operation is not permitted for the default type, nor for any User Type that has existing users. After a User Type has been deleted, it cannot be used as the type for new users, and it no longer counts against the limit of 10 User Types.
+Deletes a User Type permanently. This operation isn't permitted for the default type or for any User Type that has existing users. After a User Type has been deleted, it can't be used as the type for new users, and it no longer counts against the limit of 10 User Types.
```

Comment:
past tense to present tense

.......After you delete a User Type, it can't be used as the type for new users, and it no longer counts against the limit of 10 User Types.